archinstall/lib/disk/device_model.py

```diff
@@ -1102,13 +1424,24 @@ def _clean_field(name: str, clean_type: CleanType) -> str:
 			return name.replace('_percentage', '%').replace('_', '-')
 
 
-def _fetch_lsblk_info(dev_path: Optional[Union[Path, str]] = None) -> List[LsblkInfo]:
+def _fetch_lsblk_info(
+	dev_path: Optional[Union[Path, str]] = None,
+	reverse: bool = False,
+	full_dev_path: bool = False,
+	retry: int = 3
```

Comment:
This was not merged from master correctly. The retry variable was removed.

Reply:
PR to update this https://github.com/archlinux/archinstall/pull/2463